### PR TITLE
Update build2 0.16 -> 0.17

### PR DESCRIPTION
**Pull request by b8u**
there are some errors from `sbopkglint`:
```
Running test: 70-tmp_path...                                                                                                                                                                                                                                                                                                                                                                                                            
--- found files with TMP (/tmp/SBo/package-*) baked in:                                                                                                                                                                                                                                                                                                                                                                                 
./usr/doc/build2-0.17.0/build2.SlackBuild          << it's just a comment                                                                                                                                                                                                                                                                                                                                                                                     
./usr/lib64/libbuild2-cli-0.17-0.17.so                                                                                                                                                                                                                                                                                                                                                                                                  
./usr/lib64/libbuild2-0.17.so                                                                                                                                                                                                                                                                                                                                                                                                           
./usr/lib64/libbutl-pkg-config-0.17.so                                                                                                                                                                                                                                                                                                                                                                                                  
./usr/lib64/libbuild2-bin-0.17-0.17.so                                                                                                                                                                                                                                                                                                                                                                                                  
./usr/lib64/libbuild2-cxx-0.17-0.17.so                                                                                                                                                                                                                                                                                                                                                                                                  
./usr/lib64/libbutl-0.17.so                                                                                                                                                                                                                                                                                                                                                                                                             
./usr/lib64/libbuild2-cc-0.17-0.17.so                                                                                                                                                                                                                                                                                                                                                                                                   
./usr/lib64/libbuild2-kconfig-0.17-0.3.so                                                                                                                                                                                                                                                                                                                                                                                               
./usr/lib64/libbutl-odb-0.17.so                                                                                                                                                                                                                                                                                                                                                                                                         
./usr/lib64/libbuild2-version-0.17-0.17.so                                                                                                                                                                                                                                                                                                                                                                                              
./usr/lib64/libbuild2-in-0.17-0.17.so                                                                                                                                                                                                                                                                                                                                                                                                   
./usr/lib64/libbuild2-bash-0.17-0.17.so                                                                                                                                                                                                                                                                                                                                                                                                 
./usr/lib64/libbuild2-c-0.17-0.17.so                                                                                                                                                                                                                                                                                                                                                                                                    
./usr/lib64/libbuild2-autoconf-0.17-0.3.so                                                                                                                                                                                                                                                                                                                                                                                              
./usr/lib64/libbpkg-0.17.so                                                                                                                                                                                                                                                                                                                                                                                                             
./usr/bin/b                                                                                                                                                                                                                                                                                                                                                                                                                             
./usr/bin/bpkg                                                                                                                                                                                                                                                                                                                                                                                                                          
./usr/bin/bdep                                                                                                                                                                                                                                                                                                                                                                                                                          
FAILED                         
```

I've checked all ELFs and there are no `/tmp/SBo/package-*` in the ELF symbols. Not sure what does grep find...

### CODE REVIEW

**Comment by b8u** [comment]
I need help with the actions. My repo is not shallowed and I'm not sure why it fails

**Comment by willysr** [comment]
@sbo-bot: build development/build2

**Comment by willysr** [comment]
@aclemons need help to see what is the real error message

**Comment by b8u** [comment]
> @aclemons need help to see what is the real error message

Could it be due to execution timeout?

**Comment by aclemons** [comment]
> @aclemons need help to see what is the real error message

Sorry missed this ping. Will check soon when after work.

**Comment by aclemons** [comment]
@sbo-bot: build i586 development/build2

**Comment by aclemons** [comment]
We are hitting https://github.com/aclemons/slackbuilds-ci/issues/4

**Comment by aclemons** [comment]
I will post the detailed error when I can access it.

File: development/build2/build2.SlackBuild [hunk 1]
```diff
@@ -83,19 +79,21 @@ find -L . \
  \( -perm 666 -o -perm 664 -o -perm 640 -o -perm 600 -o -perm 444 \
   -o -perm 440 -o -perm 400 \) -exec chmod 644 {} \;
 
+# Fix chroot folder (to avoid having /tmp/SBo/package-* in the resulting files)
```
Comment (aclemons) [inline development/build2/build2.SlackBuild:82]: It looks like something is not quite right with your patch. At the end it is trying to run `b` which is part of this package. It fails there.

```
install bdep/legal{AUTHORS} -> ../package-build2/usr/share/doc/bdep/
install bdep/legal{LEGAL} -> ../package-build2/usr/share/doc/bdep/
install bdep/manifest{manifest} -> ../package-build2/usr/share/doc/bdep/
+ command -v b
```

You might want to check how arch handle it:

https://gitlab.archlinux.org/archlinux/packaging/packages/build2/-/blob/main/PKGBUILD

Comment (aclemons) [reply development/build2/build2.SlackBuild:82]: Did you test this locally? It fails in the same place for me just checking out this branch and running the slackbuild.

**Pull request closed without merging**